lib/cmds/space_cmds/environment_cmds/use.js

```diff
@@ -0,0 +1,88 @@
+import inquirer from 'inquirer'
+import { createManagementClient } from '../../../utils/contentful-clients'
+
+import { getContext, setContext, storeRuntimeConfig } from '../../../context'
+import { assertLoggedIn } from '../../../utils/assertions'
+import { handleAsyncError as handle } from '../../../utils/async'
+import { success } from '../../../utils/log'
+import paginate from '../../../utils/pagination'
+import { highlightStyle } from '../../../utils/styles'
+
+export const command = 'use'
+
+export const desc = 'Sets the default environment which every command will use when the --environment-id option is skipped.'
+
+export const aliases = ['u']
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful space environment use')
+    .option('environment-id', {
+      alias: 'e',
+      describe: 'ID of the Environment within the currently active Space to use for other commands'
+    })
+    .epilog('Copyright 2018 Contentful, this is a BETA release')
+}
+
+function showSuccess (space, environment) {
+  success(`Now using Environment ${highlightStyle(environment.name)} (${highlightStyle(environment.sys.id)}) in Space ${highlightStyle(space.name)} (${highlightStyle(space.sys.id)}) when the \`--environment-id\` option is missing.`)
+}
+
+export async function environmentUse (argv) {
+  await assertLoggedIn()
+
+  const { cmaToken, activeSpaceId } = await getContext()
+  const environmentId = argv.environmentId
+
+  const client = await createManagementClient({
+    accessToken: cmaToken
+  })
+
+  const space = await client.getSpace(activeSpaceId)
```

Comment:
Sometimes the `activeSpaceId` is not there, we should tell the user instead of showing him a 404 error.
I suggest
1. Add an optional `--space-id` 
2. if `--space-id` is specified you take it and get the environment from there
3. if not you show an error to the user tellig him to pick a space

Reply:
let's go with 3